styles/_dropdown.scss

```diff
@@ -0,0 +1,57 @@
+.dropdown__options {
```

Comment:
Use SCSS nesting to show that `__options` is an element that belongs to the block called `.dropdown`

```scss
.dropdown {
  &__options {
    // Stuff
  }
}
```